src/pages/settings/Subscription/CardSection/utils.ts

```diff
@@ -1,9 +1,140 @@
+import {fromUnixTime} from 'date-fns';
+import * as Expensicons from '@components/Icon/Expensicons';
+import * as Illustrations from '@components/Icon/Illustrations';
+import DateUtils from '@libs/DateUtils';
+import type {Phrase, PhraseParameters} from '@libs/Localize';
 import * as SubscriptionUtils from '@libs/SubscriptionUtils';
+import CONST from '@src/CONST';
+import type {TranslationPaths} from '@src/languages/types';
+import type {Fund} from '@src/types/onyx';
+import type IconAsset from '@src/types/utils/IconAsset';
+
+type BillingStatusResult = {
```

Comment:
Should we make it more strict by adding non-optional props? or atleast add `type-fest` `requireAtleastone` rules?

We should then set the return type as `BillingStatusResult | undefined` to handle `default {}` scenario.

Reply:
Great suggestion, adjusted